iModelCore/ECDb/LiveSync.md

```diff
@@ -0,0 +1,54 @@
+# No lock schema synchronization across multiple briefcases
+
+No lock does not mean lock free. It just mean that briefcases can make changes to schema without requiring them to push there changes.
```

Comment:
it means that there's no need to hold the *schema* lock.

"their", not "there"

Reply:
I rewrote it.